parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -237,7 +237,7 @@ impl pallet_authorship::Config for Runtime {
     type FindAuthor = pallet_session::FindAccountFromAuthorIndex<Self, Aura>;
     type UncleGenerations = UncleGenerations;
     type FilterUncle = ();
-    type EventHandler = ();
+    type EventHandler = (CollatorSelection,);
```

Comment:
Does it need to be inside a tuple?

Reply:
No but this is consistent with the other runtimes